### PR TITLE
Create Ada_Drivers_Library-vs-bb_runtimes.md

### PR DESCRIPTION
**Pull request by Fabien-Chouteau**

### CODE REVIEW

File: docs/Ada_Drivers_Library-vs-bb_runtimes.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+(interrupt controller, one timer, some CPU registers), but it won't touch the
+other peripherals (I2C, SPI, GPIOs, Ethernet, etc.).
+
+`Ada_Drivers_Library` on the other side is a library for firmware development
+in Ada. It provides drivers for remaining peripherals on the system as well
+drivers for external components and also middleware services. Note that
+`Ada_Drivers_Library` may rely on Ada run-time features but it does not meant
+that the run-time has to be from the `bb-runtimes` repository, there are other
+implementation possible depending on the architecture.
+
```
Comment (pat-rogers) [inline docs/Ada_Drivers_Library-vs-bb_runtimes.md]: The limit?

File: docs/Ada_Drivers_Library-vs-bb_runtimes.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+The limit between the two projects is the Ada language standard.
+
+Even if for most projects `Ada_Drivers_Library` and `bb-runtimes` will be used
+together, in the end the two emain independent. You can use the run-times with
```
Comment (pat-rogers) [inline docs/Ada_Drivers_Library-vs-bb_runtimes.md]: emain => remain

File: docs/Ada_Drivers_Library-vs-bb_runtimes.md (hunk 1 above)
Comment (Fabien-Chouteau) [inline docs/Ada_Drivers_Library-vs-bb_runtimes.md]: What do you mean? :)

Comment (Fabien-Chouteau) [inline docs/Ada_Drivers_Library-vs-bb_runtimes.md]: OK I see: `The limit between the two projects`, that must be Franglish. Maybe `The border` or `The boundary`?

File: docs/Ada_Drivers_Library-vs-bb_runtimes.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+We understand that there might be some confusion between the two repositories
+`Ada_Drivers_Library` and `bb-runtimes`. Who does what? What should go where?
+This page will try to provide and answer.
```
Comment (simonjwright) [inline docs/Ada_Drivers_Library-vs-bb_runtimes.md]: to provide **an** answer

Comment (Fabien-Chouteau) [reply docs/Ada_Drivers_Library-vs-bb_runtimes.md]: Now fixed. Thanks Simon!

**Pull request merged**